templates/single-tapestry.php

```diff
@@ -3,6 +3,40 @@
 /*
  Template Name: Tapestry Page Template
  */
+/**
```

Comment:
I basically swap the order of function calls here so that Vue.js could use the `wpApiSettings` data provided by Wordpress.